app/assets/stylesheets/all/questions.css.scss

```diff
@@ -102,3 +102,70 @@ ul.tags {
   max-height: 93px;
   overflow: auto;
 }
+
+
+// TODO: Ensure styles apply without increased specificity
+#questions-header.questions-header {
+
+  span {
+    margin-right: 10px;
+  }
+
+  a {
+    i {
+    color: inherit;
+    }
+  }
+}
+.questions-list {
+  padding-left: 0;
+
+  li {
+    list-style-position: inside;
+  }
+
+  .question {
```

Comment:
I've decided to use mixins for reusable styles. It keeps things DRY. Would that be okay?